tests/langchain_chat_sample.py

```diff
@@ -154,7 +154,7 @@ def format_docs(docs):
 #         "status_code": "UNSET"
 #     },
 #     "attributes": {
-#         "workflow_context_properties.session_id": "0x4fa6d91d1f2a4bdbb7a1287d90ec4a16",
+#         "context.session_id": "0x4fa6d91d1f2a4bdbb7a1287d90ec4a16",
```

Comment:
as context is a first class object in telemetry, it can confuse the users as json path to fetch would be context.<property name>.  My suggestion would be to use either 'invocation' or not to add any prefix

Reply:
- from perspective of capturing and querying these attributes in prompt API, should be fine either way
  - if the span of these attributes is fixed, like root span for eg, it would be easier for prompt API implementation
  - since these attributes are inserted in all spans, including root span, so prompt API implementation should be fine
- some prefix might be helpful to determine if the attribute came from framework vs custom attribute inserted by user, though not sure if this is needed
  -  some possible prefix alternatives could be: state, session, trace

Reply:
changed to session.{property_name}